app/serializers/api/v1/location_time_series_values_serializer.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class LocationTimeSeriesValuesSerializer < ActiveModel::Serializer
+      attribute :indicator_id
+      attribute :values
+
+      def indicator_id
+        object.first
+      end
+
+      def values
+        object.second.map do |tsv|
+          {
+            year: tsv.year,
+            value: tsv.value,
+            scenario_id: tsv.scenario_id
```

Comment:
Could we skip this scenario_id if we filter by scenario?

Reply:
done!